app/views/user_orders/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Profile Orders</h1>
+
+<section class="orders-index">
+  <% @user.orders.each do |order| %>
```

Comment:
Think we could use current_user just for consistency as well - so we wouldn't have to pass in @user from the controller to the view